js/src/sdk/models/backendClient.ts

```diff
@@ -86,5 +89,6 @@ export class BackendClient {
     });
 
     setAxiosClientConfig(axiosClient.instance);
+    this.instance = axiosClient.instance;
```

Comment:
Redundant assignment of `this.instance`. This is already assigned on line 39. Consider removing this duplicate assignment or add a comment explaining why it's needed twice.